openml/flows/sklearn_converter.py

```diff
@@ -342,6 +359,8 @@ def _extract_information_from_model(model):
                     cr_value = OrderedDict()
                     cr_value['key'] = identifier
                     cr_value['step_name'] = identifier
+                    if len(sub_component_tuple) == 3:
+                        cr_value['param_1'] = sub_component_tuple[2]
```

Comment:
Should we maybe rename `param_1` into `argument_1` to as this is (in my opinion) rather an argument to that specific step of the pipeline than a parameter (or hyperparameter).

Reply:
Sure